chain/indexer/src/lib.rs

```diff
@@ -44,6 +44,8 @@ pub struct InitConfigArgs {
     pub download_config: bool,
     /// Specify a custom download URL for the config file.
     pub download_config_url: Option<String>,
+    /// Specify the boot nodes to bootstrap the network
+    pub boot_nodes: Option<String>,
```

Comment:
Maybe it's worth to put a new arg after the last existing? Especially in `init_configs` function signature to make breaking change a little bit easier?

Reply:
I have put it in front, as logically this is kind tied tothe download_config flags, as well. It seemed weird to me to put it after max_gas_burnt_view for that reason :)